### DISCUSSION TITLE
Use Bun to watch for typescript errors, like `tsc --watch --noEmit`

### DISCUSSION THREAD

**Discussion by giovannivocale-molar** [oven-sh/bun#5395] [General]
In my next.js app, I keep watching for typescript error by running `tsc --noemit --watch`.

Can I replace `tsc` with `bun`? What command would I run then?

Thanks

**Comment by negativems** [comment]
This would be great

**Comment by digitalextremist** [comment]
Still using `tsc` in this case nearly 2 years later @giovannivocale-molar?
